### PR TITLE
feature(alternator): dataintegrity=true to the 3h sanity test

### PR DESCRIPTION
**Pull request by fruch**
now that we have figured the way to use dataintegrity,
and make the code to collect it's stats,
it's about time to start using it

### CODE REVIEW

**Comment by fgelcer** [comment]
@fruch , resolve conflicts

File: sdcm/sct_events.py [hunk 1]
```diff
@@ -185,7 +185,7 @@ def formatted_timestamp(self):
 
     def publish(self, guaranteed=True):
         if guaranteed:
-            return EVENTS_PROCESSES['MainDevice'].publish_event_guaranteed(self)
+            return EVENTS_PROCESSES['MainDevice'].publish_event_guaranteed(self, reuse_socket=False)
```
Comment (bentsi) [inline sdcm/sct_events.py]: this issue was addressed in #1757

Comment (fruch) [reply sdcm/sct_events.py]: I know, but I'm trying to get this branch working.
and it keep failing on this issue, I'll remove it before merging.

**Comment by fruch** [comment]
I'm happy with this one, passing:
https://jenkins.scylladb.com/job/scylla-staging/job/fruch/job/longevity/job/longevity-alternator-3h-test/35

File: test-cases/longevity/longevity-alternator-3h.yaml [hunk 2]
```diff
@@ -1,22 +1,27 @@
 test_duration: 180
 prepare_write_cmd: [
-  "bin/ycsb load dynamodb -P workloads/workloada -threads 250 -p recordcount=20971520 -p fieldcount=10 -p fieldlength=512", ]
+  "bin/ycsb load dynamodb -P workloads/workloadc -threads 80 -p recordcount=20971520 -p fieldcount=10 -p fieldlength=512 -p dataintegrity=true", ]
```
Comment (fruch) [inline test-cases/longevity/longevity-alternator-3h.yaml]: @roydahan 
image

what is the avg. CPU we expect on a basic sanity, we should aim for ~50% ? (don't remember why I've lower it, I was trying to understand why the dataintegrity was failing)

I'll put it back up a notch

File: sdcm/sct_events.py [hunk 3]
```diff
@@ -638,6 +638,9 @@ def _load_from_raw_alert(self,  # pylint: disable=too-many-arguments,invalid-nam
             self.alert_name = self.labels.get('alertname', '')
         else:
             self.alert_name = ''
+        severity = self.labels.get('severity')
```
Comment (bentsi) [inline sdcm/sct_events.py]: per our discussion, the default should be warning and custom configured where we explicitly set severity we should use it

**Comment by bentsi** [comment]
@fruch please address comments per our discussion and fix conflicts

File: sdcm/sct_events.py (hunk 3 above)
Comment (fruch) [reply sdcm/sct_events.py]: fixed

**Comment by fruch** [comment]
> @fruch please address comments per our discussion and fix conflicts

conflict fixed, and fixed what we talked about

from my POV, this is good to be merged

**Pull request merged**